requirements.txt

```diff
@@ -0,0 +1,6 @@
+pandas~=1.3.2
```

Comment:
`pandas~=1.3.2` means `any version like 1.3.2.*`
usually better to write keep only major and minor versions fixed (aka `pandas~=1.3`)